cupyx/scipy/sparse/coo.py

```diff
@@ -319,7 +320,16 @@ def tocsc(self, copy=False):
             cupyx.scipy.sparse.csc_matrix: Converted matrix.
 
         """
-        return self.T.tocsr().T
```

Comment:
Leaving a note to my future self: This change is in line with SciPy that the conversion functions should be made symmetric (in this case `tocsr` vs `tocsc`) whenever a low-level library call is available to carry out the task. In this case, this change avoids two (unnecessary) constructor calls from `.T`.